src/python-common/ceph/deployment/service_spec.py

```diff
@@ -461,6 +462,10 @@ def _cls(cls: Type[ServiceSpecT], service_type: str) -> Type[ServiceSpecT]:
             'node-exporter': MonitoringSpec,
             'prometheus': MonitoringSpec,
             'snmp-gateway': SNMPGatewaySpec,
+            'elasticsearch': TracingSpec,
+            'jaeger-agent': TracingSpec,
+            'jaeger-collector': TracingSpec,
+            'jaeger-query': TracingSpec,
```

Comment:
well. can you re-use the monitoringspec?

Reply:
no, as I wrote below, we need special fields